pandas/core/frame.py

```diff
@@ -3828,8 +3828,8 @@ def rename(self, *args, **kwargs):
         copy : boolean, default True
             Also copy underlying data
         inplace : boolean, default False
-            Whether to return a new DataFrame. If True then value of copy is
-            ignored.
+            Modify the DataFrame in place (do not create a new object).
+            If True then value of copy is ignored.
```

Comment:
I don't think the new sentence is technically correct. The `inplace` parameter is misleading, and in many cases the `DataFrame` is not modified in place, but creating a copy that is assigned to the same variable.

Can you take a look at the parameter `inplace` of other methods and see what is being used there. It'd be nice to unify them actually (not in this PR).